api/src/pcapi/alembic/versions/20230713T111742_9f5e01a79b73_init_categories_tags_offre_lieux.py

```diff
@@ -0,0 +1,33 @@
+"""init default categories on tags offres & lieux
```

Comment:
(détail) in English ;) 

```suggestion
"""init default categories on offer/venue tags
```